Src/library/store/interface/db_control.e

```diff
@@ -75,14 +75,14 @@ feature -- Basic operations
 		do
 			if not is_ok then
 				fixme ("Unicode support for output.")
-				io.error.putstring ("EiffelStore Error")
+				io.error.put_string ("EiffelStore Error")
 				if error_code /= 0 then
-					io.error.putstring (" <")
-					io.error.putint (error_code)
-					io.error.putchar ('>')
+					io.error.put_string (" <")
+					io.error.put_integer (error_code)
+					io.error.put_character ('>')
 				end
-				io.error.putstring (": ")
-				io.error.putstring (error_message_32.as_string_8)
+				io.error.put_string (": ")
+				io.error.put_string ({UTF_CONVERTER}.utf_32_string_to_utf_8_string_8 (error_message_32))
```

Comment:
yes, but here, I just updated code to remove obsolete calls without introducing a new dependencies.
If one day, we have the console output supporting encoding directly from EiffelBase, then ok.
For now, if we want to fix something, we should rather remove any console output from a library... and use an observer pattern, and it would be up to the application to "write" in console, file, or any vision widget, ...

So I will not change this now.